sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/tests/TimeSeriesInsightsQuerySeriesTests.cs

```diff
@@ -0,0 +1,217 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using FluentAssertions;
+using Microsoft.Azure.Devices.Client;
+using NUnit.Framework;
+
+namespace Azure.IoT.TimeSeriesInsights.Tests
+{
+    [LiveOnly]
+    public class TimeSeriesInsightsQuerySeriesTests : E2eTestBase
+    {
+        private static readonly TimeSpan s_retryDelay = TimeSpan.FromSeconds(30);
+
+        private const int MaxNumberOfRetries = 10;
+        private const string Humidity = "Humidity";
```

Comment:
Do you intend to use QueryTestsHelper here as well?

Reply:
Yup. Will update.